### PR TITLE
fixed err in 01 action.ts

### PR DESCRIPTION
**Pull request by web-pirate**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/receiving#contribution-guideline)

### CODE REVIEW

File: src/store/modules/return/actions.ts [hunk 1]
```diff
@@ -102,7 +102,7 @@ const actions: ActionTree<ReturnState, RootState> = {
     } catch (err) {
       showToast(translate('Something went wrong'));
       console.error("error", err);
-      return Promise.reject(new Error(err))
+      return Promise.reject(new Error("err"))
```
Comment (ymaheshwari1) [inline src/store/modules/return/actions.ts:105]: This change is not needed.

File: package.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
     "@vue/cli-plugin-router": "~4.5.0",
     "@vue/cli-plugin-typescript": "~4.5.0",
     "@vue/cli-plugin-unit-jest": "~4.5.0",
-    "@vue/cli-service": "~4.5.0",
+    "@vue/cli-service": "4.5.19",
```
Comment (ymaheshwari1) [inline package.json:49]: Can you add some information about this change?

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (web-pirate) [reply package.json:49]: Sure

Comment (web-pirate) [reply package.json:49]: It was done when I downloaded the node.js 14.10 version.

**Comment by ymaheshwari1** [comment]
Closing as will be handled in a different way.

**Pull request closed without merging**